### ISSUE TITLE
Discovery: LiaScript research

### ISSUE THREAD

**Issue by kristenjernigan** [usds/ditap-curriculum-update#72]
Objective
- Research LiaScript as a potential alternative to using Storyline Articulate for creating content

Open Questions
- Support for SCORM?
- What features of Rise/content creation do we use that LiaScript does not support?

Approach
- Explore product features
- Test content creation

Outcome
- Determine if this product would work for DITAP Refresh
- Conversation/call w/LiaScript developer
- Report on findings

Additional Information
- https://liascript.github.io/

**Comment by cruzaprild** [comment]
@kristenjernigan Hi! I believe you have this on your radar, but could you summarize your findings in a report and add it to our external ditap folder for future reference? That way, we can close this ticket out. Thank you!

**Comment by kristenjernigan** [comment]
@cruzaprild Hi April!  I have added my f[indings](https://docs.google.com/document/d/1Wqk-5P94Wvv5lKLdGrxYDqCuIogTvJSrZXSIYTy57uI/edit?usp=sharing) to the DITAP external folder. 

Please let me know if anything needs to be changed/updated.
